src/System.Threading.Tasks/tests/Task/TaskFromAsyncTest2.cs

```diff
@@ -1,394 +0,0 @@
-            TaskFromAsyncTest test = new TaskFromAsyncTest(parameters);
-            test.RealRun();
-        }
-
-        [Fact]
-        public static void TaskFromAsyncTest66()
-        {
-            TestParameters parameters = new TestParameters(API.APM_T, TaskType.Task, TaskType.Task, OverloadChoice.None, ErrorCase.None);
-            TaskFromAsyncTest test = new TaskFromAsyncTest(parameters);
-            test.RealRun();
-        }
-
-
-        [Fact]
-        public static void TaskFromAsyncTest69()
-        {
-            TestParameters parameters = new TestParameters(API.APM_T, TaskType.Task, TaskType.Task, OverloadChoice.None, ErrorCase.Throwing);
-            TaskFromAsyncTest test = new TaskFromAsyncTest(parameters);
-            test.RealRun();
-        }
-
-        [Fact]
-        public static void TaskFromAsyncTest70()
-        {
-            TestParameters parameters = new TestParameters(API.APM_T, TaskType.Task, TaskType.Task, OverloadChoice.WithTaskOption, ErrorCase.None);
-            TaskFromAsyncTest test = new TaskFromAsyncTest(parameters);
-            test.RealRun();
-        }
-
-        [Fact]
-        public static void TaskFromAsyncTest71()
-        {
-            TestParameters parameters = new TestParameters(API.APM, TaskType.Task, TaskType.Task, OverloadChoice.WithTaskOption, ErrorCase.None);
-            TaskFromAsyncTest test = new TaskFromAsyncTest(parameters);
-            test.RealRun();
-        }
-
-        [Fact]
-        public static void TaskFromAsyncTest72()
-        {
-            TestParameters parameters = new TestParameters(API.IAsyncResult, TaskType.Task, TaskType.Task, OverloadChoice.None, ErrorCase.None);
-            TaskFromAsyncTest test = new TaskFromAsyncTest(parameters);
-            test.RealRun();
-        }
-
-
-        [Fact]
-        public static void TaskFromAsyncTest74()
-        {
-            TestParameters parameters = new TestParameters(API.IAsyncResult, TaskType.Task, TaskType.Task, OverloadChoice.None, ErrorCase.NullEnd);
-            TaskFromAsyncTest test = new TaskFromAsyncTest(parameters);
-            test.RealRun();
-        }
-        [Fact]
-        public static void TaskFromAsyncTest75()
-        {
-            TestParameters parameters = new TestParameters(API.IAsyncResult, TaskType.Task, TaskType.Task, OverloadChoice.None, ErrorCase.Throwing);
-            TaskFromAsyncTest test = new TaskFromAsyncTest(parameters);
-            test.RealRun();
-        }
-
-        [Fact]
-        public static void TaskFromAsyncTest77()
-        {
-            TestParameters parameters = new TestParameters(API.IAsyncResult, TaskType.Task, TaskType.Task, OverloadChoice.WithTaskOption, ErrorCase.None);
-            TaskFromAsyncTest test = new TaskFromAsyncTest(parameters);
-            test.RealRun();
-        }
-
-        #endregion
-
-    }
-}
```

Comment:
No ,they all fail apparently because they all lead to `System.Action.BeginInvoke`. I guess that's why they are pulled out of `TaskFromAsyncTest.cs`. If we'll never support BeginInvoke I guess we can go ahead and delete this file.